sklearn/linear_model/_coordinate_descent.py

```diff
@@ -2078,6 +2083,7 @@ def _is_multitask(self):
 
     def __sklearn_tags__(self):
         tags = super().__sklearn_tags__()
+        tags.input_tags.sparse = True
```

Comment:
As mentioned somewhere else, it make smore sense to defined next to the `fit` so in `LinearModelCV`.

Reply:
done in 705c414